simple_qa_rnn/.gitignore

```diff
@@ -1,4 +1,8 @@
-data/
+data/*
+data/glove/
 resources/
 saved_checkpoints/
 __pycache__/
+relation_prediction/*cache*/*
+.idea/*
```

Comment:
`.idea/` and `.DS_Store` are for all models - they should be in the root .gitignore.